components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounter.kt

```diff
@@ -33,11 +35,13 @@ open class TabCounter @JvmOverloads constructor(
         setCount(INTERNAL_COUNT)
 
         context.obtainStyledAttributes(attrs, R.styleable.TabCounter, defStyle, 0).apply {
-            val counterColor = getColor(
-                R.styleable.TabCounter_tabCounterTintColor,
-                ContextCompat.getColor(context, R.color.mozac_ui_tabcounter_default_tint)
-            )
-            setColor(counterColor)
+            val counterColor = getColorStateList(
+                R.styleable.TabCounter_tabCounterTintColor
+            ) ?: ContextCompat.getColorStateList(context, R.color.mozac_ui_tabcounter_default_tint)
```

Comment:
I'm not super familiar with color state lists - will this call work fine when the resource ID is only a color reference and not a state list?